### DISCUSSION TITLE
Health orb and Spec att orb being click-through cause problems at zulrah and p2 warden

### DISCUSSION THREAD

**Discussion by Cameron-K03** [runelite/runelite#17078] [Ideas]
Hi,
I am one of those poor souls who have to use Cure-Me at zulrah and all-too-often i overspam the cure button and it causes me to yeet myself into the venom clouds due to the poison orb being interactable, and then when poison is gone, the hitpoints orb is not interactable, so you click through it, onto the screen somewhere.

Similarly, at wardens when you are swapping to dragon dagger to spec the core, I sometimes click the spec attack orb to enable special attacks too quickly, before the dragon dagger registers, and it causes me to yeet myself across the room away from the core. When the dagger is equipped, I can toggle special attacks like normal, but without it or another special attack weapon equipped, I can click through. 
I have not found a plugin or setting anywhere that affects this, so I ask here, can anyone please make a setting for me to enable/disable this feature maybe in the minimaps plugin or so?

New to github, so apologies if this isnt the right subdirectory.

<img width="72" alt="image" src="https://github.com/runelite/runelite/assets/146551489/da3e8283-a92f-40fc-9281-3358e7394703">
<img width="87" alt="image" src="https://github.com/runelite/runelite/assets/146551489/f7fc502f-6f74-4968-869b-499bdfba9a21">

**Comment by Macweese** [comment]
Letting you know that a plugin named "Click minimap orbs" is now on the plugin-hub and does what you describe.

**Comment by Cameron-K03** [comment]
Thank you so much 🙏🙏🙏
